packages/components/src/vue/components/inflections/inflections-subtables-wide.vue

```diff
@@ -73,4 +73,9 @@ export default {
   .infl-prdgm-tbl__cell--label {
     font-weight: 700;
   }
+
+  .infl-prdgm-tbl a.infl-prdgm-tbl__cell-reflink {
```

Comment:
can we also add these styles to the "Back to Main" link?

Reply:
Fixed styles - also added hover styles for both - sub and supp tables

Reply:
thanks!